Add support for MantisBT version 2.x.x

adgroupalex opened the pull request.

adgroupalex committed:
Add support for MantisBT version 2.x.x

vboctor commented on GoogleAnalytics/GoogleAnalytics.php:
```diff
@@ -18,7 +18,7 @@ function register() {
 		$this->page = 'config';
 
-		$this->version = "1.1";
+		$this->version = "1.2";
```
I suggest changing the version to 2.0.

adgroupalex replied:
Done.

vboctor commented on GoogleAnalytics/pages/config.php:
```diff
@@ -13,17 +13,41 @@
 
-html_page_top1( plugin_lang_get( 'title' ) );
-html_page_top2();
+//html_page_top1( plugin_lang_get( 'title' ) );
```
Delete dead code.

adgroupalex replied:
Done.

vboctor commented on GoogleAnalytics/pages/config.php:
```diff
@@ -13,17 +13,41 @@
 
 access_ensure_global_level( ADMINISTRATOR );
```
Should we remove the admin access level check and use the manage_plugin_threshold that was referenced below?

adgroupalex replied:
Done.

vboctor reviewed (changes requested):
Thanks @adgroupalex for your contribution.  I have added some minor comments.

Did you make sure there is no conflicts between integrating Google Analytics and CSP rules that is part of the recent releases? i.e. no errors in Javascript console and google analytics receiving the updates.

adgroupalex committed:
Changing version of plugin to 2.0

adgroupalex committed:
Use the manage_plugin_threshold instead admin access level check.

Also delete dead code.

adgroupalex committed:
Delete dead code

adgroupalex commented:
Please help me, how can I check conflicts between integrating Google Analytics and CSP rules (errors in Javascript console and google analytics receiving the updates)?
I use this plug-in a few years with mantis 1.2-1.3 and upgraded now to mantis 2.10. 
I did not have visible errors.

adgroupalex committed:
Bug fix with string constants

"Copy/paste" error from other plugin.

adgroupalex committed:
Small change for better visible script in html page

adgroupalex committed:
Translate plugin to Russian language

adgroupalex commented:
I found information about debugging Google Analytics. I will verify how Google Analytics (analytics.js) work.

adgroupalex committed:
Updated script code. Resolved conflict with CSP rules (updated CSP 1.0 rules).

Code contains some dead code with my investigation how work CSP 2.0 and how resolve confict based on CSP 2.0 rules (nonce). Today CSP 2.0 used about 78% brosers instead 94% CSP 1.0 (https://caniuse.com/#search=Content%20Security%20Policy). But I don't delete dead code because, I think, this help in future.
Also if html page contains "script-src 'unsafe-inline'" in header and "nonce=" parameter in code then unsafe-inline will be ignored. This can happen because of the conflict between several plugins or the core.

adgroupalex commented:
I resolved conflict between Google Analytics script and CSP rules. Now plugin work properly.
